pkg/binding/interfaces.go

```diff
@@ -54,19 +77,6 @@ type Message interface {
 	// intermediate Event.
 	Binary(BinaryEncoder) error
```

Comment:
I am not sure what `Binary` does. It might need a verb in there, like `ToBinary`

Reply:
`Binary` and `Structured` are the two methods to access to the message structure. I can argue that `To[X]` is not suitable, because it hints that returns a value of type X, while here it doesn't return nothing (except the `error`) because it mutates the state of `BinaryEncoder` you pass